modules/install/pages/ipv6-setup.adoc

```diff
@@ -48,3 +68,14 @@ After you install the Couchbase Server application, proceed as follows:
 . Delete `~/Library/Application\ Support/Couchbase/var/lib/couchbase/config/config.dat`.
 . Restart the `couchbase-server` service (this starts Couchbase Server in IPv6 mode).
 To check, point a supported web browser to  `http://[::1]:8091`, which is the IPv6 address and port number on which Couchbase Server should be running.
+
+=== Version 5.5.3
```

Comment:
Perhaps "Version 5.5.3 and higher 5.5.* versions" or something similar to answer the question what should be used for 5.5.4